openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -122,6 +122,20 @@ instances:
     #
     # send_monotonic_counter: true
 
+    ## @param send_distribution_counts - boolean - optional - default: false
+    ## If set to true, sends histograms counters when also sending Datadog distribution metrics.
+    ## Only relevant if send_distribution_buckets is set also to true - counters are always sent if not using
+    ## distribution metrics
+    #
+    # send_distribution_counts: false
+
+    ## @param send_distribution_sums - boolean - optional - default: false
+    ## If set to true, sends histograms sums when also sending Datadog distribution metrics.
+    ## Only relevant if send_distribution_buckets is set also to true - sums are always sent if not using distribution
+    ## metrics
+    #
+    # send_distribution_sums: false
+
```

Comment:
Since you are adding more config options to `openmetrics`, you need to add them via modifying the `spec.yaml` file for `openmetrics`, which uses the `instance/openmetrics` template found here: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml. 

You should add your config values in that `yaml` file, and then run `ddev validate config --sync` so that other integrations that use openmetrics will also have these config parameters added to their own `conf.yaml.example`.